### ISSUE TITLE
Op BatchNormWithGlobalNormalization is deprecated.

### ISSUE THREAD

**Issue by zthompson2350** [llSourcell/tensorflow_image_classifier#9]
After several issues, I managed to get the directory linked with the docker container. I got through all the steps to then realize that something had gone wrong and I would need to start over. I created a new directory with the images to be classified, as well as the label_image.py file that I got from this repository. Opened a new docker container and went through the steps... git pull, and the image_retraining. After it had finished the retraining I ran the label_image.py. This time, it seemed like it was in fact going to run at first... but instead I got the following.

2017-04-22 23:38:17.666525: W tensorflow/core/framework/op_def_util.cc:332] Op BatchNormWithGlobalNormalization is deprecated. It will cease to work in GraphDef version 9. Use tf.nn.batch_normalization().
blue cones (score = 0.09615)

So, my image classifier is looking for different colored cones, and it should be noted that the cone I used was orange, not blue. So it did have a low confidence value on it being blue, which is great. Except I'm not getting values back for the other colors of cones that I'm looking for.

I've tried running this on the images with which I've retrained the tensorflow, blue cones come back with an orange score, orange cones come back with a blue score, and yellow cones come back with a blue score as well.

On top of this, there's that error above where it gives the confidence score on it being a blue cone. I believe this has something to do with inception being outdated, but I wouldn't know how to go about fixing it at this point.

**Comment by zthompson2350** [comment]
To get it to spit out a prediction for each cone, I had to move the print statement inside the for loop at the end. I don't see why it's outside of it to start with. I still feel as though the issue for which this topic is named should be looked at.
